config/wallet-config.json

```diff
@@ -114,23 +114,46 @@
   "tokensEnabledByDefault": [
     "DOGGOTOTHEMOON",
     "RSICGENESISRUNE",
-    "PUPSWORLDPEACE",
+    "PUPSWORLDPEAC",
```

Comment:
@fbwoolf I see a typo here PUPSWORLDPEAC, should be PUPSWORLDPEACE,

Reply:
the rest LGTM

Reply:
Ok, thx, I saw that too but thought it was intentionally changed, haha.